idl/fpga.idol

```diff
@@ -0,0 +1,133 @@
+// FPGA server IPC interface
+
+Interface(
+    name: "Fpga",
+    ops: {
+        "device_enabled": (
+            doc: "Return true if the FPGA is enabled, false otherwise",
+            reply: Result(
+                ok: "u8",
```

Comment:
Idol supports `bool` variables (as of https://github.com/oxidecomputer/idolatry/pull/14); this also applies to `set_device_enable` / `application_enabled` / `set_application_enabled`